server/service/handler.go

```diff
@@ -255,17 +228,8 @@ func makeKitHandlers(e FleetEndpoints, opts []kithttp.ServerOption) *fleetHandle
 		ForgotPassword:                        newServer(e.ForgotPassword, decodeForgotPasswordRequest),
 		ResetPassword:                         newServer(e.ResetPassword, decodeResetPasswordRequest),
 		Me:                                    newServer(e.Me, decodeNoParamsRequest),
-		ChangePassword:                        newServer(e.ChangePassword, decodeChangePasswordRequest),
 		CreateUserWithInvite:                  newServer(e.CreateUserWithInvite, decodeCreateUserRequest),
```

Comment:
Just curious, why not migrating this one `CreateUserWithInvite`.

Reply:
`CreateUserWithInvite` is a bit different as it's not an `authenticatedUser` endpoint, I'm keeping those special cases for the end (same for `ResetPassword`, `VerifyInvite`, etc.). Eventually they will all get migrated.